pkg/storegateway/indexheader/header.go

```diff
@@ -45,7 +46,7 @@ type Reader interface {
 
 	// LookupSymbol returns string based on given reference.
 	// Error is return if the symbol can't be found.
-	LookupSymbol(o uint32) (string, error)
+	LookupSymbol(ctx context.Context, o uint32) (string, error)
```

Comment:
Do we need the context on this one? I can't see where we need it.

Reply:
Gone now.